rs/crypto/internal/crypto_lib/bls12_381/type/src/lib.rs

```diff
@@ -1689,24 +1699,53 @@ macro_rules! declare_compute_mul2_table_inline {
         We build up the table incrementally using additions and doubling, to
```

Comment:
We should update the docs to mention the shift and what the table entries correspond to.